packages/fx-core/src/plugins/resource/bot/resources/messages.ts

```diff
@@ -107,4 +109,10 @@ export class Messages {
   public static readonly RecreateTheProject = "Please recreate the project.";
   public static readonly CheckCommandOutputAndTryToFixIt =
     "Please check the command output and try to fix it.";
+  public static readonly DeleteExistingBotChannelRegistration =
+    "Please delete existing azure bot channel registrations.";
```

Comment:
It is proper none, better to use "Azure Bot Channels Registration", you can confirm with PM.
![image](https://user-images.githubusercontent.com/72963627/118751759-7c005800-b894-11eb-8ab3-6f1071babe5f.png)